src/server/locals.js

```diff
@@ -0,0 +1,161 @@
+/* globals Map */
+var inspector = require('inspector');
+var async = require('async');
+
+function Locals(config) {
```

Comment:
defining classes like this the old school JS way has the problem of possible function call without instantiation, i.e. missing the `new` keyword by just calling `Locals(…)` as a function. To prevent this from doing harm, a pattern emerged for checking how a constructor is called like this:

```javascript
function Locals(config) {
  if (!(this instanceof Locals)) {
    return new Locals(config);
  }
  // rest of constructor logic
}
```

Reply:
Added.